core/trino-main/src/main/java/io/trino/operator/OperatorFactories.java

```diff
@@ -56,32 +61,62 @@ OperatorFactory probeOuterJoin(
             PartitioningSpillerFactory partitioningSpillerFactory,
             BlockTypeOperators blockTypeOperators);
 
-    OperatorFactory lookupOuterJoin(
-            int operatorId,
-            PlanNodeId planNodeId,
-            JoinBridgeManager<?> lookupSourceFactory,
-            boolean waitForBuild,
-            boolean hasFilter,
-            boolean spillingEnabled,
-            List<Type> probeTypes,
-            List<Integer> probeJoinChannel,
-            OptionalInt probeHashChannel,
-            Optional<List<Integer>> probeOutputChannels,
-            OptionalInt totalOperatorsCount,
-            PartitioningSpillerFactory partitioningSpillerFactory,
-            BlockTypeOperators blockTypeOperators);
+    public static class JoinOperatorType
```

Comment:
`Modifier 'public' is redundant for interface members`
`Modifier 'static' is redundant for inner classes of interfaces`

Reply:
Thanks. I wonder why InteliJ did not notice this.

Reply:
I've InteliJ Ultimate - and it's not that I'm an eagle.